### ISSUE TITLE
Is this ready?

### ISSUE THREAD

**Issue by renpytom** [LuqueDaniel/vscode-language-renpy#67]
I've been using vscode-language-renpy exclusively when I edit Ren'Py, both as part of games and as part of the engine. It's the best tool for edting Ren'Py that I know of.

Is this extension mature enough to be installed automatically by Ren'Py? What I'm thinking is having Ren'Py automatically download vs code from the Microsoft servers, then tell code to install vscode-language-renpy, and then executing code to edit a project. If that makes sense, then I'd hope to add this is 7.5/8.0.

**Comment by LuqueDaniel** [comment]
Thank you. In the last few months and thanks to @rdurfee and the other contributors, the extension has improved a lot. Since we released version 2.0 several patches have been released to fix bugs and stabilize features. I think it's mature enough.

What do you think @rdurfee?

**Comment by rdurfee** [comment]
Yes, I agree. I think it’s stable as well. There’s always room for improvement of course, but I think it’s more than ready to go wide release.

**Comment by renpytom** [comment]
Okay, I'll be working on a new system for installing VS Code and this extension.

Let me know if either/both of you want invites to the dev channel on the Ren'Py discord.

**Comment by LuqueDaniel** [comment]
Great. My discord is @kyplim

**Comment by renpytom** [comment]
@LuqueDaniel Could you message me on discord? My account is renpytom#9751 .

**Comment by renpytom** [comment]
I've finished the integration with Ren'Py, and while it needs testing, it should be available in the current nightly build of Ren'Py.
